sendgrid.com.domain-auth.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId":"sendgrid.com",
+  "providerName":"Twilio SendGrid",
+  "serviceId":"domain-auth",
+  "serviceName":"Twilio SendGrid Domain Authentication",
+  "version": 1,
+  "logoUrl":"https://sendgrid.com/brand/sg-twilio/SG_Twilio_Lockup_RGBx1.png",
+  "syncPubKeyDomain": "domainconnect.sendgrid.net",
+  "syncRedirectDomain": "sendgrid.com",
+  "records":[
+    {
+      "type": "CNAME",
+      "host": "%host%",
+      "pointsTo": "%POINTS_TO%",
+      "ttl": 1800
+    },
+    {
+      "type": "CNAME",
+      "host": "%SUBDOMAIN_DKIM1%",
+      "pointsTo": "%POINTS_TO_DKIM1%",
```

Comment:
When using `host` parameter all subdomains will be re-rooted at the applied subdomain.
So if the intention is for `host=sub123` and `domain=example.com` to get a record like `s1._domainkey.example.com` please put `%SUBDOMAIN_DKIM1%.%domain%.` as a record value (note the dot at the end for FQDN).
If the intention is to generate `s1._domainkey.sub123.example.com` then it's OK like it is. AFAIK DKIM requires the record on the subdomain, but it may be specific to your implementation.